tests/ui/test_ui_tools.py

```diff
@@ -1680,21 +1680,32 @@ def test_main_view_generates_EDITED_label(self, mocker,
             assert label[1][1] == 7
 
     @pytest.mark.parametrize('to_vary_in_each_message, realm_editing_allowed,\
-                             expect_editing_to_succeed', [
-        ({'sender_id': 2, 'timestamp': 45}, True, False),
-        ({'sender_id': 1, 'timestamp': 1}, True, False),
-        ({'sender_id': 1, 'timestamp': 45}, False, False),
-        ({'sender_id': 1, 'timestamp': 45}, True, True),
-    ], ids=['msg_sent_by_other_user',
-            'time_limit_esceeded',
-            'editing_not_allowed',
-            'all_conditions_met'])
+                             expect_editing_to_succeed, key', [
+        pytest.param(to_vary_in_each_message, realm_editing_allowed,
+                     expect_editing_to_succeed, key, id='-'.join([_id, key]))
+        for (to_vary_in_each_message, realm_editing_allowed,
+             expect_editing_to_succeed, keys, _id) in [
+                ({'sender_id': 2, 'timestamp': 45}, True, False,
+                    keys_for_command('EDIT_MESSAGE'),
+                    'msg_sent_by_other_user'),
+                ({'sender_id': 1, 'timestamp': 1}, True, False,
+                    keys_for_command('EDIT_MESSAGE'),
+                    'time_limit_exceeded'),
+                ({'sender_id': 1, 'timestamp': 45}, False, False,
+                    keys_for_command('EDIT_MESSAGE'),
+                    'editing_not_allowed'),
+                ({'sender_id': 1, 'timestamp': 45}, True, True,
+                    keys_for_command('EDIT_MESSAGE'),
```

Comment:
As in the other cases, let's reduce this duplication of `keys_for_command` with the same parameter.